test/trento_web/plugs/api_redirector_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule TrentoWeb.Plugs.ApiRedirectorTest do
+  use TrentoWeb.ConnCase, async: true
+  use Plug.Test
+
+  alias TrentoWeb.Plugs.ApiRedirector
+
+  defmodule StubRouter do
+    def __routes__ do
+      [
+        %{verb: :get, path: "/api/v1/test"},
+        %{verb: :get, path: "/api/v1/some-resource/12345"}
```

Comment:
I thought we would have the parametrized route here `/api/v1/some-resource/:resourceId`, otherwise the two tests are basically the same.

However, I could be missing something, so up to you deciding what's best.

Reply:
We don't have all the routing chain in the test, but you spotted a bug, thanks man, reverting to draft

Reply:
solved that bug for subrouting, thx